JumpscaleCore/clients/threebot/ThreebotClient.py

```diff
@@ -95,12 +95,10 @@ def verifykey_obj(self):
             self._verifykey_obj = VerifyKey(verifykey)
         return self._verifykey_obj
 
-    # def auth(self, bot_id):
-    #     nacl_cl = j.data.nacl.get()
-    #     nacl_cl._load_privatekey()
-    #     signing_key = nacl.signing.SigningKey(nacl_cl.privkey.encode())
-    #     epoch = str(j.data.time.epoch)
-    #     signed_message = signing_key.sign(epoch.encode())
-    #     cmd = "auth {} {} {}".format(bot_id, epoch, signed_message)
-    #     res = self._redis.execute_command(cmd)
-    #     return res
+    def test_auth(self, bot_id):
```

Comment:
are these the tests for the auth? can you rename the function to `test` and follow the same approach we use for other tests. Also, may be just use `j,tools.threebot.me.default.tname` instead of having the `bot_id` instance, since it's the name of the local threebot from what I can see.